elide-core/src/main/java/com/yahoo/elide/core/RequestScopedTransaction.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core;
```

Comment:
I agree with this comment. It does seem unnecessary to place the dirty objects in with `requestScope`. That said, would this cause a breaking change to the interface? I believe it would since we have no good way of defaulting that behavior afaict.